subxt/src/tx/tx_progress.rs

```diff
@@ -100,16 +100,16 @@ where
     ///
     /// **Note:** transaction statuses like `Invalid`/`Usurped`/`Dropped` indicate with some
     /// probability that the transaction will not make it into a block but there is no guarantee
-    /// that this is true. In those cases the stream is closed however, so you currently have no
-    /// way to find out if they finally made it into a block or not.
+    /// that this is true. In those cases you have to re-subscribe to the extrinsic/create a new
+    /// TxProgress repeatedly until the extrinsic is finalized.
```

Comment:
```suggestion
    /// that this is true. In those cases the stream is closed however, so you currently have no
    /// way to find out if they finally made it into a block or not.
```

Just to revert this since turns out you can't re-subscribe :)